external-crates/move/crates/move-cli/src/base/test.rs

```diff
@@ -66,6 +66,14 @@ pub struct Test {
     /// Collect coverage information for later use with the various `move coverage` subcommands. Currently supported only in debug builds.
     #[clap(long = "coverage")]
     pub compute_coverage: bool,
+
+    /// The seed to use for the randomness generator.
+    #[clap(name = "seed", long = "seed")]
+    pub seed: Option<u64>,
+
+    /// The number of iterations to run each test that uses generated values.
+    #[clap(name = "num-iters", long = "num-iters", default_value = "10")]
+    pub num_iters: u64,
```

Comment:
Good questions:
1. I don't think we should warn since you will be running with a seed to recreate a test failure -- and we stop iteration at the first test failure. So in the presence of a `seed` `num-iters` isn't really that meaningful.
2. I think that makes sense to me. Will add that in!